### ISSUE TITLE
Gitlab self hosted projects from nested groups are ignored

### ISSUE THREAD

**Issue by yangm97** [renovatebot/pro#81]
### Which Renovate are you using? x the right box:

- [ ] Renovate Open Source CLI
- [ ] WhiteSource Renovate App
- [x] Renovate Pro
- [ ] Other (please write below)

### Which platform are you using? x the right box:

- [ ] GitHub.com
- [ ] GitHub Enterprise
- [ ] GitLab.com
- [x] GitLab self-hosted
- [ ] Bitbucket Server
- [ ] Bitbucket Cloud
- [ ] Azure DevOps

### Have you checked the logs? Don't forget to include them if relevant

```
ubuntu@gitlab:~$ docker service logs -f renovate_renovate
renovate_renovate.1.2f1x6um7e0x2@gitlab    |  INFO: Initializing server container                   
renovate_renovate.1.2f1x6um7e0x2@gitlab    |  INFO: Running in commercial license mode
renovate_renovate.1.2f1x6um7e0x2@gitlab    |  INFO: License name: MYIO            
renovate_renovate.1.2f1x6um7e0x2@gitlab    |  INFO: Trying gitlab platform: REDACTED
renovate_renovate.1.2f1x6um7e0x2@gitlab    |  INFO: Using GitLab endpoint: REDACTED
renovate_renovate.1.2f1x6um7e0x2@gitlab    |  INFO: Authenticated as GitLab user: renovate-bot      
renovate_renovate.1.2f1x6um7e0x2@gitlab    |  INFO: jobScheduler running                                                                          
renovate_renovate.1.2f1x6um7e0x2@gitlab    |  INFO: Discovered 17 repositories                    
renovate_renovate.1.2f1x6um7e0x2@gitlab    |  INFO: Repository list from scheduler         
renovate_renovate.1.2f1x6um7e0x2@gitlab    |        "repos": [                                               
renovate_renovate.1.2f1x6um7e0x2@gitlab    |          "myio/renovate", 
[redacted repo list]                                
renovate_renovate.1.2f1x6um7e0x2@gitlab    |        ]                                                
renovate_renovate.1.2f1x6um7e0x2@gitlab    |  INFO: Using cron schedule: 0 * * * *
renovate_renovate.1.2f1x6um7e0x2@gitlab    |  INFO: Job scheduler started         
renovate_renovate.1.2f1x6um7e0x2@gitlab    |  INFO: Job worker started               
renovate_renovate.1.2f1x6um7e0x2@gitlab    |  INFO: Initialized GitLab webhook handler
renovate_renovate.1.2f1x6um7e0x2@gitlab    |  INFO: Webhook listener bound to port 8080 
```
later
```
renovate_renovate.1.2f1x6um7e0x2@gitlab    |  INFO: Repository has changed during renovation - aborting (repository=myio/renovate)
```

### What would you like to do?

I've added the Renovate user to the team and it does detect the repos, but won't create the onboarding PR. If I run the non-pro renovate from cli (with --dry-run) it says it would create PRs right away while the pro version won't create PRs even if you trigger it through the webhook (for instance, by removing the bot from the team and adding to a single repository (because team adds don't trigger the autodiscover feature it seems)), it always says the repository has changed during the renovation, even though it didn't. Bellow is everything I used to deploy renovate:

`docker-compose.yml`

```
version: '3.7'

configs:
  config.js:
    name: config.js_${HASH:-undefined}
    file: config.js

volumes:
  tmp:

services:
  renovate:
    image: renovate/pro:0.18.0
    environment:
      - ACCEPT_AGREEMENT=y
      - LICENSE_MODE=commercial
      - LICENSE_NAME
      - RENOVATE_PLATFORM=gitlab
      - RENOVATE_ENDPOINT
      - RENOVATE_TOKEN # Personal Access Token for bot account
      - GITHUB_COM_TOKEN # Personal Access Token for github.com (used for retrieving changelogs)
      - WEBHOOK_SECRET # Optional, default to 'renovate'
      - SCHEDULER_CRON # Optional, defaults to '0 * * * *' (hourly)
    ports:
      - "8080:8080"
    volumes:
      - "tmp:/tmp" # Use a volume for this as the cache will cause a lot of churn
    configs:
      - source: config.js
        target: /usr/src/webapp/config.js
```

`config.js`

```
module.exports = {
  autodiscover: 'true',
  logLevel: 'info',
  requireConfig: true,
  onboarding: true,
  onboardingConfig: {
    extends: [
      'config:base',
      'preview:dockerCompose'
    ],
    separateMinorPatch: true,
    packageRules: [
      {
        updateTypes: ['patch'],
        automerge: true,
        automergeType: 'branch'
      },
      {
        updateTypes: ['major', 'minor'],
        automerge: false
      }
    ]
  }
}
```

`.gitlab-ci.yml`

```
stages:
  - deploy

variables:
  HASH: ${CI_COMMIT_SHORT_SHA}

deploy:
  image: docker:latest
  tags:
    - renovate
  stage: deploy
  script:
    - docker stack deploy renovate -c docker-compose.yml
```

**Comment by yangm97** [comment]
Is the source code for Renovate Pro available somewhere? Thinking about sending a PR for this, doesn't look too hard to fix.

**Comment by rarkins** [comment]
Thanks for the offer, but Renovate Pro is currently built out of a monorepo sharing code with some non-OSS projects, so it's not possible. I fixed it and published [0.18.5](https://github.com/renovatebot/pro/releases/tag/0.18.5) to Docker Hub instead.
